libs/sdk-backend-tiger/src/convertors/fromBackend/afm/dimensions.ts

```diff
@@ -94,10 +101,23 @@ function transformDimension(
     };
 }
 
+/**
+ * Compute mapping from attribute identifiers to all ITotalDescriptors corresponding to that attribute.
+ */
+function getAttrTotals(def: IExecutionDefinition): AttrTotals {
+    const attrTotalss: AttrTotals[] = def.dimensions.map((dim) => {
```

Comment:
typo: attrTotalss

Reply:
That's not a typo. I use this naming pattern to indicate lists of lists :-) If you find it inappropriate I can change to... ?

Reply:
how about totalsInDimsByType?

Reply:
Well, that's incorrect (it's mapping from attributes to totals/ITotalDescriptors) while not indicating the extra list level at all. I can change the name to just `attrTotals` but personally I find it incomplete.

Reply:
apologies i misread. so we can have totalsInDimsByAttr? because there is a list with per-dim mappings of attr id to all totals?

look, this is in the end small stuff, but I anticipate that the `ss` for anyone who reads this in the UI codebase will look like typo anyway so the intended meaning will be lost on them.

Reply:
Eventually changed to `attrTotals`.